src/components/quiz/UserDetailsForm.scss

```
@@ -0,0 +1,24 @@
+@import "../../constants.scss";
+*{
+  box-sizing: border-box;
+  background-color:$secondary-light;
+}
+h2{
```

Gaps between, and space before brace (do to all in this file)
```suggestion
}

h2 {
```